charts/stable/archivebox/docs/installation_guide.md

```diff
@@ -2,46 +2,57 @@
 title: Installation Guide
 ---
 
-## SCALE App
+## Configuration
 
-1. to setup archivebox:
+- Configure an initial admin username and password you want to log in within the Web UI (only applied on first setup run).
 
-- `App Configuration > Admin Username & Admin Password`
-  Configure an initial admin username and password you want to log in with in the Web UI (only applied on first setup run).
-  <img width="500" alt="Screenshot of example Admin Username & Password configuration" src="https://github.com/pirate/truecharts/assets/511499/fcd60d72-2339-4eed-bb13-5c1cd7134236">
+  ```yaml
+  values:
+    config:
+      admin_username: "admin"
+      admin_password: "changeme"
+  ```
 
-- `Networking and Services > Service's Port(s) Configuration > Port`
-  Configure the port you want to access the ArchiveBox web interface on, or use the pre-populated default.
-  <img width="500" alt="Screenshot of example Archivebox webserver port configuration" src="https://github.com/pirate/truecharts/assets/511499/f5083399-64e0-4088-8187-dfd92851eb34">
+- If needed, it is possible to set up a persitent storage for archived data.
+
+  Example configuration using NFS:
+
+  ```yaml
+  values:
+    persistence:
+      archive:
+        enabled: true
+        mountPath: /data/archive
+        type: nfs
+        path: /mnt/Tank/apps/archivebox
+        server: ${NAS_IP}
+  ```
+
+  Make sure whatever filesystem the data is located on supports FSYNC and doesn't squash permissions.
+
+  NFS servers may need to set <code>no_root_squash</code>
+  - On TrueNas Scale this equals to <code>Maproot User = root</code> and <code>Maproot Group = wheel</code> under **Advanced Options**
```

Comment:
Please dont include scale specific stuff in the guides

Reply:
OK